PowerPointLabs/PowerPointLabs/ActionFramework/CaptionsLab/RemoveCaptions/RemoveCaptionsActionHandler.cs

```diff
@@ -13,7 +13,7 @@ protected override void ExecuteAction(string ribbonId)
         {
             //TODO: This needs to improved to stop using global variables
```

Comment:
@initialshl  the TODO comment says to stop using global variables, is IsRemoveCaptionsEnabled which is a public static bool variable still considered a global variable?